### PR TITLE
chore(connectors): updated runtime execution information

### PR DESCRIPTION
**Pull request by igpetrov**
## Description

chore(connectors): updated runtime execution information

## When should this change go live?

Anytime

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [x] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/self-managed/platform-deployment/manual.md [hunk 1]
```diff
@@ -168,18 +168,58 @@ To update Tasklist versions, visit the [guide to update Tasklist](../../componen
 
 ## Run Connectors
 
-The [Connector runtime environment](https://repo1.maven.org/maven2/io/camunda/spring-zeebe-connector-runtime) picks up outbound Connectors available on the `classpath` automatically.
-It uses the default configuration specified by a Connector through its `@OutboundConnector` annotation.
+### Bundle
 
-To run the [REST Connector](https://search.maven.org/artifact/io.camunda.connector/connector-http-json) with the runtime environment, execute the following command:
+Bundle includes runtime with all Camunda official connectors.
```
Comment (christinaausley) [inline docs/self-managed/platform-deployment/manual.md:173]: Do you just mean Camunda's available Connectors here?

Comment (igpetrov) [reply docs/self-managed/platform-deployment/manual.md:173]: That's right, all Camunda's available Connectors

Comment (christinaausley) [reply docs/self-managed/platform-deployment/manual.md]: Adjusted in latest commit 👍

**Review by christinaausley — APPROVED** [review]
Looks good to me 👍

**Pull request merged**